pytest_mh/utils/fs.py

```diff
@@ -757,3 +763,23 @@ def sed(self, command: str, path: str, args: list[str] | None = None) -> Process
         self.logger.info(f"Running sed {command} on {path}")
         args = args if args else []
         return self.host.conn.exec(["sed", *args, command, path], log_level=ProcessLogLevel.Error)
+
+    def restorecon(self, path: str) -> bool:
+        """
+        Restore selinux context on a file or directory.
+        Does nothing when restorecon is not present on os without selinux.
+
+        :param path: File or directory where changes will happen
+        :type path: str
+        :return: True if restorecon succeeded
+        :rtype: bool
```

Comment:
I am concerned that this function will return `False` if `restorecon` is not present in the system, as this may be seen as an error by the caller. I wonder if this shouldn't be an `int/enum` type, which will help us detect different types of scenarios.

Reply:
The idea is that this will be noop on distros without selinux.
If we need to make sure that restorecon exists the fs utility can already do that before calling this.
We do not need to complicate this one.

Reply:
Current implementation isn't a noop as any caller checking the return value for this method where SELinux isn't present would get a `False` value and this could be interpreted as a failure. If we are to return a boolean, then the non-SELinux case should return `True`